### PR TITLE
Scripts to launch VS and VS Code using local SDK

### PR DESCRIPTION
**Pull request by jander-msft**
These are the same scripts from https://github.com/dotnet/aspnetcore with a minor adjustment for the startvs.cmd script to automatically open the dotnet-monitor.sln file.

### CODE REVIEW

**Comment by wiktork** [comment]
It might be nice to also add this functionality to generate-dev-sln.ps1.

**Review by wiktork — APPROVED** [review]

File: startvscode.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    exit 1
+fi
+
+if [[ $1 == "" ]]; then
+  code .
+else
+  code $1
+fi
+
+exit 1
```
[the remark this answers is not in the document]
Comment (jander-msft) [reply startvscode.sh:29]: I'll remove that

**Pull request merged**